src/profiler.h

```diff
@@ -68,8 +68,8 @@ class Profiler {
     Engine* _alloc_engine;
     int _event_mask;
 
-    time_t _start_time;
-    time_t _stop_time;
```

Comment:
There are no more usages of `time_t` in profiler.h. Does it mean `#include <time.h>` can be moved from profiler.h to .cpp?